doc/main.html

```diff
@@ -8,11 +8,11 @@
   <script type="module" src="../smpte.js"></script>
   <meta itemprop="pubType" content="AG" />
   <meta itemprop="pubNumber" content="26" />
-  <!-- <meta itemprop="pubPart" content="XX" /> -->
+  <!-- <meta itemprop="pubPart" content="XX" / -->
```

Comment:
Probably unintended change

Reply:
yep, rolled back in https://github.com/SMPTE/html-pub/pull/315/commits/0584aee517056f0768191f9c38d3369bb6514bdf